microk8s-resources/actions/disable.host-ip.sh

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
+
+set -e
+
+echo "Disabling host-ip"
+
+if [ -f "${SNAP_DATA}/var/lock/host-ip-enabled" ]
+then
+  IP_ADDRESS=$(<"${SNAP_DATA}/var/lock/host-ip-enabled")
+  sudo ifconfig lo:17 $IP_ADDRESS down
```

Comment:
what is the 17 in lo:17?

Reply:
just a random number, not to conflict with any other existing lo:interface-number which a user might have added. Change it to lo:1?

Reply:
changed to lo:1

Reply:
Shouldn't we try to select a number that will not conflict with the existing user setup?

Reply:
Could you call the interface lo:microk8s instead of trying to do arithmetic with the lo: interfaces?

Reply:
done